### PR TITLE
Move nodesReverseTentativeTime to a vector

### PR DESCRIPTION
**Pull request by greenscientist**
Apply the same optimisation than nodesTentativeTime

### CODE REVIEW

**Comment by greenscientist** [comment]
We see an extra 15%-20% speed up for the summary call in the "forward" case and a good 50% speed up for the "reverse" case. 
Confirmed same results in my test cases

File: connection_scan_algorithm/src/reverse_calculation.cpp [hunk 1]
```diff
@@ -147,8 +139,7 @@ namespace TrRouting
 
                 if (footpathTravelTime <= parameters.getMaxTransferWalkingTravelTimeSeconds())
                 {                  
-                  if ((trite == nodesReverseTentativeTime.end()) || //TODO Unclear if it's equivalent than previous code
```
Comment (tahini) [inline connection_scan_algorithm/src/reverse_calculation.cpp:150]: Is this now equivalent to previous code? ;-)

**Review by tahini — APPROVED** [review]
Thanks

File: connection_scan_algorithm/src/reverse_calculation.cpp (hunk 1 above)
Comment (greenscientist) [reply connection_scan_algorithm/src/reverse_calculation.cpp:150]: Yeah, I went back in history and put back the structure we had before :)

**Pull request merged**